test/test_pipeline.cpp

```diff
@@ -81,7 +101,7 @@ TEST_F(NVFuserTest, Pipeline_CUDA) {
       " PipelineVal representing Val T0_g[ iS0{i0}, iS1{i2} ] on stage " +
       std::to_string(stage0.unique_id) +
       "\n"
-      " PipelineVal representing Val T4_g[ iS6{i13}, iS7{i14}, iS8{i15} ] on stage " +
+      " PipelineVal representing Val T4_g[ iS6{i15}, iS7{i16}, iS8{i17} ] on stage " +
```

Comment:
These changes were necessitated by the addition of an attribute in `ReductionOp`. A `Val*` is created when each attribute is added, incrementing the scalar name counter.